jetty-server/src/test/resources/jetty-logging.properties

```diff
@@ -1,5 +1,5 @@
 org.eclipse.jetty.util.log.class=org.eclipse.jetty.util.log.StdErrLog
-org.eclipse.jetty.LEVEL=WARN
+org.eclipse.jetty.LEVEL=INFO
```

Comment:
Extraneous to the subject of this PR, so this change will be hard to find in future, so IMHO better to be just a single commit by itself.